frontend/src/redux/Game.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Game, getGame } from '../api/Game';
+
+const initialState = null as Game | null;
```

Comment:
I originally tried that but also came across an error, I think it may have something to do [with this](https://stackoverflow.com/questions/63037105/why-is-there-an-error-on-createslice-when-the-initial-state-is-null)? Not sure if that's a TypeScript or Redux or our bug